spring-batch-docs/asciidoc/spring-batch-integration.adoc

```diff
@@ -315,58 +313,57 @@ represented by the job parameter "input.file.name":
 ----
 
 
-The main points of interest here are injecting the value of
+The main points of interest in the preceding example are injecting the value of
 `#{jobParameters['input.file.name']}`
 as the Resource property value and setting the `ItemReader` bean
-to be of __Step scope__ to take advantage of
-the late binding support which allows access to the
+to have __Step scope__, Setting the bean to have Step scope takes advantage of
+the late binding support, which allows access to the
 `jobParameters` variable.
 
 
 [[availableAttributesOfTheJobLaunchingGateway]]
 === Available Attributes of the Job-Launching Gateway
 
-* `id` Identifies the underlying Spring bean definition, which is an instance of either:
+The job-launching gateway has the following attributes that you can set to control a job:
+
+* `id`: Identifies the underlying Spring bean definition, which is an instance of either:
 ** `EventDrivenConsumer`
 ** `PollingConsumer`
-
-The exact implementation depends on whether the component's input channel is a:
-`SubscribableChannel` or `PollableChannel`
-
-* `auto-startup` Boolean flag to indicate that the endpoint should start automatically on
+(The exact implementation depends on whether the component's input channel is a
+`SubscribableChannel` or `PollableChannel`.)
+* `auto-startup`: Boolean flag to indicate that the endpoint should start automatically on
 startup. The default is __true__.
-* `request-channel` The input `MessageChannel` of this endpoint.
-* `reply-channel` `Message Channel` to which the resulting `JobExecution` payload will be sent.
-* `reply-timeout` Allows you to specify how long this gateway will wait for the reply message
+* `request-channel`: The input `MessageChannel` of this endpoint.
+* `reply-channel`: `Message Channel` to which the resulting `JobExecution` payload is sent.
```

Comment:
Remove space between Message and Channel

Reply:
Yes, please. Thanks.